DataDefinitions/Ship.cs

```diff
@@ -291,8 +291,9 @@ public string starsystem
 
         public decimal? fueltankcapacity { get; set; } // Core capacity
         public decimal? fueltanktotalcapacity { get; set; } // Capacity including additional tanks
-        public decimal maxfuel { get; set; }
-        public decimal maxjump { get; set; }
+        public decimal maxjumprange { get; set; }
+        public decimal maxfuelperjump { get; set; }
```

Comment:
Do we need to replace the current properties? This could be a breaking change for some users and I'm not sure there's much distinction (except that the numbers will be more precise now).

Reply:
Only thing it’s breaking is my script, which I have changed.

Reply:
It'll also break any modified versions the script that other users may have written, won't it?

Reply:
It’s aligns better with FDEV naming convention.

Reply:
Then consider preserving the old variables for compatibility? eg...
```cs
        [JsonIgnore, Obsolete("Please use maxjumprange instead")]
        public decimal maxjump => maxjumprange;

        public decimal maxjumprange { get; set; }
```
Or at minimum, noting the change in the change log so that users aren't taken entirely by surprise.

Reply:
Changed.